web/explorer/src/components/DescriptionPanel.vue

```diff
@@ -16,13 +16,22 @@
             <p class="text-700 text-sm m-0">New to capa? Follow these quick steps to get started:</p>
             <p class="text-700 text-sm mb-3"></p>
             <ol class="text-left text-sm pl-4 mb-3">
-                <li class="mb-2"><a href="https://github.com/mandiant/capa/blob/master/doc/installation.md" class="underline">Install capa</a>, e.g.
-                <ul>
-                    <li>download the latest <a href="https://github.com/mandiant/capa/releases" class="underline">standalone executable release</a></li>
-                    <li>or run <code>$ pip install flare-capa</code></li>
-                </ul>
+                <li class="mb-2">
+                    <a href="https://github.com/mandiant/capa/blob/master/doc/installation.md" class="underline"
+                        >Install capa</a
+                    >, e.g.
+                    <ul>
+                        <li>
+                            download the latest
+                            <a href="https://github.com/mandiant/capa/releases" class="underline"
+                                >standalone executable release</a
+                            >
+                        </li>
+                        <li>or run <code>$ pip install flare-capa</code></li>
+                    </ul>
```

Comment:
I don't like this reformatting, especially the hanging `>` characters.

Reply:
https://stackoverflow.com/questions/56291245/how-do-you-stop-prettier-in-vs-code-splitting-attributes-onto-multiple-lines recommends increasing the print width

Reply:
@s-ff how do you recommend to run the linter and write the changes?

Reply:
After adding this option in `.prettier.json`, simply run `npm run fomat` to apply formatting. 


To inspect all available commands, run `npm run`.

Reply:
thanks, can you take another look now at this PR?

Reply:
Thanks, this seems to resolve the issue.